spaces/domains.json

```
@@ -472,4 +472,5 @@
   "vote.underfitted.io": "underfitted.eth",
   "snapshot.silo.finance": "silofinance.eth",
   "signal.paladin.vote": "palvote.eth"
+  "dao.futureland.io": "futuredigital.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "signal.paladin.vote": "palvote.eth",
  "dao.futureland.io": "futuredigital.eth"
```